src/main/scala/de/uhh/lt/spark/corpus/DefaultSource.scala

```diff
@@ -0,0 +1,110 @@
+package de.uhh.lt.spark.corpus
```

Comment:
this is another package which is also called corpus. it is implemented without any dependencies to josimtext. having it outside of the jst namespace tries to indicate this seperation. it seems to be unimportant where it is located currently and we might absolutely want to change that in the future. do you still want it moved?